CMakeLists.txt

```diff
@@ -44,11 +44,11 @@ set_target_properties(${IPFS_API_LIBNAME} PROPERTIES
   VERSION ${PROJECT_VERSION}
 )
 target_link_libraries(${IPFS_API_LIBNAME} ${CURL_LIBRARIES})
-
-install(TARGETS ${IPFS_API_LIBNAME} DESTINATION lib)
-install(FILES include/ipfs/client.h DESTINATION include/ipfs)
-install(FILES include/ipfs/http/transport.h DESTINATION include/ipfs/http)
-
+if(ENABLE_INSTALL)
+   install(TARGETS ${IPFS_API_LIBNAME} DESTINATION lib)
```

Comment:
3 spaces as indentation whereas the rest of the file uses 2.

Reply:
Fixed identation